templates/common/.github/workflows/bicep/azure-dev.yml

```diff
@@ -30,6 +28,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Install azd
```

Comment:
I don't think we should call this task install azd and have task name be setup azd.  Can we decide if we want setup or install?  I think it should be install...

Reply:
The `setup-$TOOLNAME` convention is fairly common, so I think we want to continue to use `azure/setup-azd` as the repository for the action name, so I guess we should change to to `Setup azd` as the "name" of the step.